app/scripts/migrations/134.ts

```diff
@@ -0,0 +1,41 @@
+import { hasProperty, isObject } from '@metamask/utils';
+import { cloneDeep } from 'lodash';
+
+type VersionedData = {
+  meta: { version: number };
+  data: Record<string, unknown>;
+};
+
+export const version = 134;
+
+/**
+ * This migration removes `providerConfig` from the network controller state.
```

Comment:
Outdated description:
```suggestion
 * This migration removes `usedNetworks` from the `AppStateController` state.
```